src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealClient.groovy

```diff
@@ -76,10 +76,10 @@ interface SurrealClient {
     @Put('/key/wave_request/{token}')
     Flux<Map<String, Object>> updateContainerRequestAsync(@Header String authorization, String token, @Body WaveContainerRecord body)
 
-    @Post('/key/wave_scan/{id}')
-    Map<String,Object> insertScanRecord(@Header String authorization, String id, @Body WaveScanRecord body)
+    @Post('/key/wave_scan')
+    Map<String,Object> insertScanRecord(@Header String authorization, @Body WaveScanRecord body)
```

Comment:
I'd like to understand why this is not working anymore?

Reply:
These API calls, where we are passing id as a separate parameter and id also present inside the object `WaveScanRecord` surrealdb is, throwing error:` result:Found 'cve-3' for the id field, but a specific record has been specified, status:ERR, time:21.875µs`
this is the resposne from surrealdb guys :
<img width="1198" alt="Screenshot 2024-01-23 at 19 02 16" src="https://github.com/seqeralabs/wave/assets/6575288/c7475023-158d-44e4-bb1d-dc41f3cd5b9e">

https://discord.com/channels/902568124350599239/970338835990974484/1198983764282441809

Reply:
I will check here for two scenarios:
1. What id does the assume in the DB when now id is passed in post?
2. What happens when an object with the same id is stored in DB again?

Reply:
We may not need to keep compatibility with the old db. See here #387

Reply:
Look good